whatsdown/src/services/servicesCrud.ts

```diff
@@ -0,0 +1,70 @@
+import axios from "axios";
```

Comment:
use
```suggestion
import axios from "../shraga/axios.ts";
```
then you wont have to add
```js
    {
      headers: {
        Authorization: `Bearer ${cookies.get(environment.accessTokenName)}`,
      },
    }
```
in every function

Reply:
so, i just remove the headers?

Reply:
the headers are removed from each individual request but they still get sent by the axios request interceptor